### PR TITLE
Csharp: Initial Lib & Webhook Verification

### LINKED ISSUE #83: [0.5] Add C# webhook verification & signing
Including deployment

### PR DESCRIPTION
**Pull request by svix-frank**
Closes #83

Assuming we should get @svix-yair to sign off on this one.

### CODE REVIEW

**Review by tasn — APPROVED** [review]
I don't know csharp, but if the tests pass, and the code looks alright, I think this can go in. :)

**Comment by tasn** [comment]
Yeah, probably wait on Yair

File: csharp/Svix/Webhook.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            {
+                key = key.Substring(prefix.Length);
+            }
+
+            this.key = Convert.FromBase64String(key); ;
+        }
+
+        public void Verify(string payload, WebHeaderCollection headers)
+        {
+            string msgId = headers.Get("svix-id");
```
Comment (svix-yair) [inline csharp/Svix/Webhook.cs:29]: nit: Move keys to const and also use in the tests

**Review by svix-yair — APPROVED** [review]
LGTM one nit comment

File: csharp/Svix/Webhook.cs (hunk 1 above)
Comment (svix-frank) [reply csharp/Svix/Webhook.cs:29]: I'll make them consts, the only question is do we want to expose these as part of the public API? The tests are in their own project so there's no way to access them there without publicly exposing afaik.

Comment (svix-frank) [reply csharp/Svix/Webhook.cs]: I made them consts but kept them private, if you think we should expose this we discuss and add them in a separate PR.

**Pull request merged**